crates/sp-domains-fraud-proof/src/runtime_interface.rs

```diff
@@ -10,13 +10,19 @@ use sp_runtime_interface::runtime_interface;
 /// Domain fraud proof related runtime interface
 #[runtime_interface]
 pub trait FraudProofRuntimeInterface {
-    fn get_invalid_domain_extrinsic_root_info(
+    /// Returns required fraud proof verification information to the runtime through host function.
+    fn get_fraud_proof_verification_info(
         &mut self,
         consensus_block_hash: H256,
         domain_id: DomainId,
```

Comment:
Are you sure these are the only arguments that will be necessary for any kind of fraud proof. I'm thinking whether they should be moved under enum or deserve to be here.